### ISSUE TITLE
[1.16.5] Crash with Create 0.3.2C

### ISSUE THREAD

**Issue by ocram9** [mrh0/createaddition#116]
**Describe the bug**
When I try to start the game with Create 0.3.2C, it crashes with the message: 

```
Failure message: Create (create) encountered an error during the sided_setup event phase
java.lang.IllegalStateException: Cannot start registration when already started!
```

This time the report blamed Create, but other times it blames this mod. Removing this mod fixes the crashing.

**To Reproduce**

Install Flywheel 0.2, Create 0.3.2C and Create Crafts & Additions 20210731a and run the game.

**Additional context**
- Mod version: 20210731a
- Forge version: 36.2.0
- Create version: 0.3.2C

Crashlog: https://gist.github.com/ocram9/b3767446459254647e14d298b8914f36

**Comment by Shotguun29** [comment]
Yeah same here
https://gist.github.com/Shotguun29/ada6002401e771bd2742a0d7ff96d86b

**Comment by mrh0** [comment]
It's a bug in Create but a temp fix is in 20210801a.
